pkg/oci/client/client.go

```diff
@@ -49,6 +45,7 @@ type Interface interface {
 	BlockStorage() BlockStorageInterface
 	FSS() FileStorageInterface
 	Identity() IdentityInterface
+	ContainerEngine() ContainerEngineInterface
```

Comment:
Is this available in the public SDK? I saw tests regarding this being removed somewhere in the 1.24 PR

Reply:
This was removed in later commits.